### PR TITLE
fix postgresql annotations

### PR DESCRIPTION
**Pull request by bgabbasov**

### CODE REVIEW

**Review by vitek — APPROVED** [review]

**Comment by vitek** [comment]
thanks!

**Review by vitek — COMMENTED** [review]
if it's optional here it also must be optional here https://github.com/yandex/yandex-taxi-testsuite/blob/06e5601f3e188ee6fd6a0c9aba9d46cbf874c1da/testsuite/databases/pgsql/discover.py#L96

File: testsuite/databases/pgsql/discover.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ class PgShardedDatabase:
 
 
 def find_schemas(
-        service_name: str, schema_dirs: List[pathlib.Path],
+        service_name: Optional[str], schema_dirs: List[pathlib.Path],
```
Comment (vitek) [inline testsuite/databases/pgsql/discover.py:70]: why do you want it to be optional?

**Review by vitek — CHANGES_REQUESTED** [review]

File: testsuite/databases/pgsql/discover.py (hunk 1 above)
Comment (bgabbasov) [reply testsuite/databases/pgsql/discover.py:70]: There are 2 reasons:

1. The documentation says that it is optional - service_name: service name used as prefix for database name if not empty, e.g. "servicename_dbname". If it is empty string, then the result will be "_dbname", and None gives expected output "dbname"
2. I have a use case when I need to pass None. I can send you a link to PR in arc.

Comment (vitek) [reply testsuite/databases/pgsql/discover.py:70]: okk, so make it optional for _find_databases_schemas as well

**Pull request merged**